clouddriver-appengine/src/main/java/com/netflix/spinnaker/clouddriver/appengine/storage/controllers/AppengineStorageController.java

```diff
@@ -28,15 +28,17 @@
 import java.util.ArrayList;
 import java.util.List;
 
+// TODO(jacobkiefer): Refactor this Controller into a common controller with injected StorageService(s) when we
+// add another storage account service. Leaving this in Appengine's scope for now.
 @Slf4j
 @RestController
-@RequestMapping("/appengine/storage")
+@RequestMapping("/storage")
```

Comment:
Should this class be moved too? To clouddriver-web

Reply:
Let's move that in a future refactor when we add another accounts source.